demos/c_api_minimal_app/README.md

```diff
@@ -0,0 +1,90 @@
+# C API inference demo (C/C++)
```

Comment:
No sphinx directives at the beginning of the file means it will not end up in demos list in docs.openvino.ai. Do we need it? @dtrawins @rasapala